rotkehlchen/rotkehlchen.py

```diff
@@ -896,27 +903,31 @@ def _query_and_populate_exchange_asset_movements(
             )
         else:
             assert isinstance(exchange, Location), 'only a location should make it here'
-            assert exchange in EXTERNAL_EXCHANGES, 'only csv supported exchanges should get here'  # noqa : E501
-            location = exchange
-            # We might have no exchange information but CSV imported information
-            self.actions_per_location['asset_movement'][location] = 0
-            location_movements = self.data.db.get_asset_movements(
-                from_ts=from_ts,
-                to_ts=to_ts,
-                location=location,
-            )
-
+            # We get here from locations that have been imported by a CSV file
+            # Ignore the location if the exchange is also connected
+            if exchange not in self.exchange_manager.connected_exchanges.keys():
+                location = exchange
+                # We might have no exchange information but CSV imported information
+                self.actions_per_location['asset_movement'][location] = 0
+                location_movements = self.data.db.get_asset_movements(
+                    from_ts=from_ts,
+                    to_ts=to_ts,
+                    location=location,
+                )
+            else:
+                location = None
```

Comment:
to separate the code section with the 2 if/elses please add a new line here
```suggestion
                location = None

```